Ghidra/Features/FunctionID/ghidra_scripts/RepackFid.java

```diff
@@ -55,11 +56,21 @@ private void copyTable(Table oldTable,PackedDBHandle newHandle) throws IOExcepti
 
 	@Override
 	protected void run() throws Exception {
-		File file = askFile("Select FID database file to repack","OK");
```

Comment:
I thought the various `ask` methods automatically pull from the script args in headless mode.  Did that "feature" not work for you here?